Engine/source/math/mPoint3.h

```diff
@@ -855,7 +859,8 @@ inline F64 Point3D::lenSquared() const
 
 inline F64 Point3D::len() const
 {
-   return mSqrtD(x*x + y*y + z*z);
+	F64 temp = x*x + y*y + z*z;
```

Comment:
A released optimized build would fix this for us, but it really isn't necessary to duplicate expressions when you can help it. Also tabs vs 3 spaces issue.
```C++
   F64 temp = x*x + y*y + z*z;
   return (temp > 0.0) ? mSqrtD(temp) : 0.0;
```